virttest/utils_misc.py

```diff
@@ -2114,9 +2114,9 @@ def get_support_machine_type(qemu_binary="/usr/libexec/qemu-kvm"):
     c = []
     for item in o[1:]:
         machine_list = re.split(r"\s+", item, 1)
-        if machine_list[0] != "none":
+        if machine_list[0] != "none" and "alias of" not in machine_list[1]:
```

Comment:
you can skip lines if it contain  word"alias" or "none" to avoid get duplicate machine types.
machine_types= [ re.split('\s+', line)[0] for line in o[1:] if not ( 'alias' in line or 'none' in line ) ]

Reply:
doc string said this function will return list, but in fact it return a tuple.

Reply:
@xutian thx, updated, could you help review again?